src/Riok.Mapperly/Descriptors/MappingBuilder/ObjectPropertyMappingBuilderContext.cs

```diff
@@ -144,28 +138,45 @@ private PropertyNullDelegateAssignmentMapping GetOrCreateNullDelegateMappingForP
         return mapping;
     }
 
+    private void InitIgnoredUnmatchedProperties(out IReadOnlyCollection<string> result, IEnumerable<string> allProperties, IEnumerable<string> mappedProperties)
```

Comment:
Any reason why this needs to have an `out` parameter instead of simply returning the collection?

Reply:
Not anymore, switched to return value.